### PR TITLE
Sidemenu

### PR DESCRIPTION
**Pull request by meshack34**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Reference the issue related to this PR as shown below. Every issue has it's own number you can find the issue numbers [here](https://github.com/italanta/italanta-apps/issues).

Fixes # (issue) - *Example*: Fixes #10, #20 NOTE: GitHub issue numbers should be prefixed with **#** to aid with issue tracking.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Screenshot (optional)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**: (optional)
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: libs/elements/layout/page-convl/src/lib/components/convl-sidemenu/convl-sidemenu.component.html [hunk 1]
```diff
@@ -1,3 +1,16 @@
+<mat-toolbar class="mat-elevation-z8">
+  <button
+    mat-icon-button
+    *ngIf="sidenav.mode === 'over'"
+    (click)="sidenav.toggle()"
+  >
+    <mat-icon *ngIf="!sidenav.opened"> menu </mat-icon>
+    <mat-icon *ngIf="sidenav.opened"> close </mat-icon>
+  </button>
+  Responsive side navigation
```
Comment (jrosseel) [inline libs/elements/layout/page-convl/src/lib/components/convl-sidemenu/convl-sidemenu.component.html:10]: What is the meaning of the text "Responsive side navigation" ?

File: libs/elements/layout/page-convl/src/lib/components/convl-sidemenu/convl-sidemenu.component.html [hunk 2]
```diff
@@ -32,7 +45,7 @@
 
       <div fxLayout="row">
         <i matListIcon class="fas fa-home list-icon"></i>
-        Home
+        main
```
Comment (jrosseel) [inline libs/elements/layout/page-convl/src/lib/components/convl-sidemenu/convl-sidemenu.component.html:48]: Please keep calling it Home in line with design

File: libs/elements/layout/page-convl/src/lib/components/convl-sidemenu/convl-sidemenu.component.html [hunk 3]
```diff
@@ -47,81 +60,7 @@
       </div>
     </div>
 
-    <!-- Section Chats -->
```
Comment (jrosseel) [inline libs/elements/layout/page-convl/src/lib/components/convl-sidemenu/convl-sidemenu.component.html:50]: Don't remove these comments for later reference

File: libs/elements/layout/page-convl/src/lib/convl-page.module.ts [hunk 4]
```diff
@@ -14,6 +14,13 @@ import { SubNavBarComponent } from './components/sub-bar/sub-navbar.component';
 import { ConvlNavbarComponent } from './components/convl-navbar/navbar.component';
 
 
+import { MatToolbarModule } from '@angular/material/toolbar';
```
Comment (jrosseel) [inline libs/elements/layout/page-convl/src/lib/convl-page.module.ts:17]: These modules are already a part of MaterialDesignModule hence don't need to be imported

**Review by jrosseel — CHANGES_REQUESTED** [review]

**Comment by meshack34** [comment]
yes, Working on it

**Pull request closed without merging**